components/next-page.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+
+    <!-- Next page button -->
+    <b-row align-h="end">
+        <b-col class="text-right" cols="4">
+
+            <!-- Optional instructions to remind user of criteria to proceed to the next page -->
+            <p class="instructions-text" v-if="!pageAccessible(currentPage)">
```

Comment:
I think we're better off being visually explicit about the steps needed to gain next page access. The tooltip is visually cleaner but it's not an obvious UI affordance - it would turn into something the user accidentally discovered.